packages/backend/server/src/modules/sync/events/events.gateway.ts

```diff
@@ -160,53 +202,138 @@ export class EventsGateway implements OnGatewayConnection, OnGatewayDisconnect {
     ).toString('base64');
     const state = Buffer.from(encodeStateVector(doc)).toString('base64');
 
-    endTimer();
     return {
       missing,
       state,
     };
   }
 
+  @SubscribeMessage('client-update-v2')
```

Comment:
maybe using same message name, but distinguish by a `version` field in the payload?

Reply:
that will make the function body too verbose, and by separating them, it will be easier to remove when the time coming(no more old clients)